diff --git a/vendor/Panda3DS/third_party/glad/CMakeLists.txt b/vendor/Panda3DS/third_party/glad/CMakeLists.txt
index b6311342..07413c1c 100644
--- a/vendor/Panda3DS/third_party/glad/CMakeLists.txt
+++ b/vendor/Panda3DS/third_party/glad/CMakeLists.txt
@@ -4,6 +4,8 @@ if(WIN32)
     set(GLAD_FILES ${GLAD_FILES} src/glad_wgl.c)
 elseif(ANDROID)
     set(GLAD_FILES ${GLAD_FILES} src/glad_egl.c)
+elseif(ORBIS)
+    set(GLAD_FILES ${GLAD_FILES} src/glad_egl.c)
 elseif(NOT APPLE)
     set(GLAD_FILES ${GLAD_FILES} src/glad_egl.c src/glad_glx.c)
 endif()
